src/test/java/com/epam/training/center/hw2/ex1/AbstractPage.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.training.center.hw2.ex1;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import java.util.concurrent.TimeUnit;
+
+public abstract class AbstractPage {
+    protected WebDriver driver;
```

Comment:
pls fromat lines: no excess empty lines and add empty lines where needed to improve readability